package ATM.States;

import ATM.RoomComponents.ATM;
import ATM.RoomComponents.Card;

public class IdleState extends ATMState {
    @Override
    public void insertCard(ATM atm, Card card) {
        System.out.println("Card is inserted");
        atm.setCurrentATMState(new HasCardState());
    }
}